libstuff/SHTTPSManager.h

```diff
@@ -15,14 +19,21 @@ class SHTTPSManager : public STCPManager {
         SData fullResponse;
         int response;
         STable values;
-        SHTTPSManager& owner;
+        SHTTPSManager& manager;
         bool isDelayedSend;
         uint64_t sentTime;
+
+        class NotLeading : public exception {
+            const char* what() {
+                return "Can't create SHTTPSManager::Transaction when not leading";
+            }
+        };
     };
 
     // Constructor/Destructor
-    SHTTPSManager();
-    SHTTPSManager(const string& pem, const string& srvCrt, const string& caCrt);
+    //SHTTPSManager();
```

Comment:
did you mean to leave this commented out default constructor in here? 

Also requiring a plugin to create an shttpsmanager feels like it some what violates the spirit of libstuff, which from the readme is meant to be used just generically for anything c++ related. For example I've made a few projects that source libstuff as an external git module and uses https managers, now thats not possible. 

Should we leave the default constructor for this use case and allow to create an shttpsmanager that doesn't care about leading?